### PR TITLE
Add Context in Host

### LINKED ISSUE #85: Add a subentry for Context in Host entry of server.xml
In server.xml, we add a subentry in Host to specify the Context of the application.
For example : 
`<Host name="localhost">
    <Context path="" docBase="..." crossContext="true"/>
</Host>`
If you're ok i'll make a PR.

### PR DESCRIPTION
**Pull request by HerveMARTIN**
Closes #85

### CODE REVIEW

**Comment by awuetz** [comment]
Hello @HerveMARTIN ,

do you know when this feature will be available?

**Comment by antoineco** [comment]
My fault, I haven't reviewed yet!

**Comment by awuetz** [comment]
Ok, you would save me a lot of headache, if this would be available soon.

**Comment by awuetz** [comment]
Hi @antoineco,

do you know when a new release will be available?
Kind regards.

**Comment by antoineco** [comment]
@awuetz reviewing right now :)

File: manifests/instance.pp [hunk 1]
```diff
@@ -233,6 +233,9 @@
   $host_unpackwars            = undef,
   $host_params                = {},
   #..................................................................................
+  # host context
+  $host_context        = undef,
```
Comment (antoineco) [inline manifests/instance.pp]: You added the parameter here but not the corresponding `concat::fragment` later in the same file.

File: manifests/init.pp [hunk 2]
```diff
@@ -259,6 +259,9 @@
   $host_unpackwars            = undef,
   $host_params                = {},
   #..................................................................................
+  # host context
+  $host_context        = undef,
```
Comment (antoineco) [inline manifests/init.pp]: Minor thing: could you please align the equal sign on the other class params?

File: manifests/instance.pp (hunk 1 above)
Comment (antoineco) [inline manifests/instance.pp]: ... and same comment regarding the equal sign alignment.

**Review by antoineco — CHANGES_REQUESTED** [review]
Please consider my comments, the rest is perfect 👍

**Comment by HerveMARTIN** [comment]
Done :)

**Review by antoineco — APPROVED** [review]

**Comment by antoineco** [comment]
Thanks Hervé !

**Comment by awuetz** [comment]
Hi @antoineco,
will you release a new version?

**Comment by HerveMARTIN** [comment]
At least @awuetz you could use a git ref to this master branch in your Puppetfile.
```puppet
mod 'aco-tomcat',
     :git => 'https://github.com/antoineco/aco-tomcat'
```

**Comment by antoineco** [comment]
Released in [v1.8.3](https://forge.puppet.com/aco/tomcat/1.8.3/readme)

**Pull request merged**